addons/hr_recruitment/wizard/applicant_refuse_reason.py

```diff
@@ -44,6 +44,12 @@ def _default_refuse_reason_id(self):
         help="send emails after that date. This date is considered as being in UTC timezone."
     )
 
+    @api.depends('refuse_reason_id')
+    def _compute_send_mail(self):
+        for wizard in self:
+            template = wizard.refuse_reason_id.template_id
+            wizard.send_mail = template and not wizard.applicant_without_email
```

Comment:
Don't you think it should depend on ``applicant_without_email`` ? 

Thanks for your work !